cvr_osc_lib/osc_interface.py

```diff
@@ -78,120 +98,401 @@ def __init__(
         self.dispatcher: Dispatcher = Dispatcher()
 
         # noinspection PyTypeChecker
+        #uncomment the following line to enable the default handler (debugging purposes only)
         # self.dispatcher.set_default_handler(osc_default_handler)
 
         self.osc_lib_ip = osc_lib_ip
         self.osc_lib_port = osc_lib_port
         self.osc_cvr_ip = osc_cvr_ip
         self.osc_cvr_port = osc_cvr_port
+        self.print_starting_messages = True
 
         self.sender = None
         self.receiver = None
 
-    def start(self, *, start_sender=True, start_receiver=True):
+    def start(self,
+              *,
+              start_sender=True,
+              start_receiver=True,
+              print_starting_messages=True):
+        """
+        Starts the OSC sender and receiver threads.
+
+        parameters
+        ----------
+        start_sender : bool, optional
+            Whether to start the sender thread or not
+        start_receiver : bool, optional
+            Whether to start the receiver thread or not
+        print_starting_messages : bool, optional
+            Whether to print the starting messages or not
+
+        returns
+        -------
+        None
+        """
+
+        self.print_starting_messages = print_starting_messages
+
         if self.sender is None and start_sender:
             self._start_sender()
 
         if self.receiver is None and start_receiver:
-            self.receiver = BlockingOSCUDPServer((self.osc_lib_ip, self.osc_lib_port), self.dispatcher)
-            print(f'Starting the OSC receiver... Will listen for messages from {self.osc_lib_ip}:{self.osc_lib_port}')
+            self.receiver = BlockingOSCUDPServer((self.osc_lib_ip, self.osc_lib_port),
+                                                 self.dispatcher)
+            self._print_starting_message('receiver')
```

Comment:
The line width should be 120 characters. This shouldn't be a line break.

Reply:
potentially fixed in [2e85b40](https://github.com/kafeijao/cvr_osc_lib_py/pull/3/commits/2e85b407f4289b74f6a18cc101e731042534353c)